ksql-parser/src/test/java/io/confluent/ksql/parser/SqlFormatterTest.java

```diff
@@ -172,7 +171,7 @@ public void setUp() {
         ORDERS_SCHEMA,
         SerdeOption.none(),
         KeyField.of(ColumnRef.withoutSource(ColumnName.of("ORDERTIME"))),
-        new MetadataTimestampExtractionPolicy(),
+        Optional.empty(),
```

Comment:
Is it worth having at least one test that has column and testing its formatted correctly?